README.md

```diff
@@ -16,7 +16,7 @@ Features include:
 Get started:
 
   * Install testify with [one line of code](#installation), or [update it with another](#staying-up-to-date)
-  * For an introduction to writing test code in Go, see https://go.dev/doc/code#Testing
+  * For an introduction to writing test code in Go using its default testing framework, see https://go.dev/doc/code#Testing
```

Comment:
I don't think that the wording is that much better. Because in fact knowing the `testing` framework is requirement to be able to use `Testify`. So this is not about `testing` being the *default* but being the *base* framework.